pybossa/api/project.py

```diff
@@ -140,8 +140,7 @@ def _update_object(self, obj):
 
     def _update_attribute(self, new, old):
         for key, value in old.info.items():
-            if not new.info.get(key):
-                new.info[key] = value
+            new.info.setdefault(key, value)
```

Comment:
this change is unrelated to reset_presented_time. it is to fix the bug where a project setting attribute whose value to be updated is to be changed from `True` to `False` cannot be done because the condition `if not new.info.get` turns out to be `True` due to which old value is retained.